plugins/flatpak/gs-flatpak.c

```diff
@@ -2736,8 +2736,11 @@ gs_flatpak_refine_runtime_for_install (GsFlatpak *self,
 }
 
 static GsAppList *
-gs_flatpak_get_list_for_install (GsFlatpak *self, GsApp *app,
-				 GCancellable *cancellable, GError **error)
+gs_flatpak_get_list_for_install_or_update (GsFlatpak *self,
+					   GsApp *app,
+					   gboolean is_update,
```

Comment:
I'd normally suggest using an `enum` here instead of a gboolean as that's usually a better thing to do, but since this is a private static function only used from 2 places, I think it's fine. Your call.

Reply:
I prefer to keep the boolean for now.